src/options/proof_options.toml

```diff
@@ -269,3 +269,11 @@ name   = "Proof"
   type       = "uint64_t"
   default    = "2"
   help       = "the lookahead depth for merging proofs at previsit"
+
+[[option]]
```

Comment:
Since it's best not to modify the API, I think we have to omit this option.

For testing purposes, I would maintain this option on a dev branch however.